docs/preview/connected-registry/quickstart-view-connected-registry-repos-and-tags.md

```diff
@@ -0,0 +1,170 @@
+- The Azure CLI commands in this article are formatted for the Bash shell. If you're using a different shell like PowerShell or Command Prompt, you may need to adjust line continuation characters or variable assignment lines accordingly. This article uses variables to minimize the amount of command editing required.
+
+## Before you begin
+
+Make sure that you have created the connected registry resource in Azure as described in the [Create connected registry using the CLI][quickstart-connected-registry-cli] quickstart guide and have a connected registry deployed on your premises as described in [Quickstart: Deploy a connected registry to an IoT Edge device](quickstart-deploy-connected-registry-iot-edge-cli.md) or [Quickstart: Deploy a connected registry to Kubernetes cluster](quickstart-deploy-connected-registry-kubernetes.md). 
+
+In this tutorial, we configure a _client token_ to list available repositories and tags. Reference [Understand access to a connected registry](overview-connected-registry-access.md) for more information on how client tokens are used for authentication with a connected registry.
+
+## Create a client token
+
+Use the following CLI command to create a token with read access to your synced repositories. We need both `content/read` and `metadata/read` permissions to list the repositories and tags in the connected registry. The below command creates a token with read access to the "hello-world" and "testing" repositories. 
+
+```azurecli
+az acr token create \
+  --name myconnectedregistry-client-token \
+  --registry mycontainerregistry001 \
+  --repository hello-world content/read metadata/read \
+  --repository testing content/read metadata/read
+```
+
+The command will return details about the newly generated token including passwords.
+
+  > [!IMPORTANT]
+  > Make sure that you save the generated passwords. Those are one-time passwords and cannot be retrieved. You can generate new passwords using the [az acr token credential generate][az-acr-token-credential-generate] command.
+
+## Update the connected registry with the client token
+
+Use the following CLI command to update the connected registry with the newly created client token:
+
+```azurecli
+az acr connected-registry update \
+  --name myconnectedregistry \
+  --registry mycontainerregistry001 \
+  --add-client-token myconnectedregistry-client-token
+```
+
+## View the repositories in a deployed connected registry
+
+Run the following commands from a machine with access to the deployed connected registry.
+
+1. Install [jq](https://stedolan.github.io/jq/), a command-line JSON processor. This utility is used to parse JSON, and is useful in parsing the response when listing repositories and tags of a connected registry. From the open SSH connection, enter following command to install jq:
+
+```
+sudo apt -y install jq
```

Comment:
Do you want to include curl and base64 used in the samples?

Reply:
Good point, added instructions for these. Moved machine config info into its own section.